### PR TITLE
test: reenable jinad tests

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: tests/integration/jinad/Dockerfiles/Dockerfile
Comment (JoanFM) [inline tests/integration/jinad/Dockerfiles/Dockerfile:1]: how does this image guarantee that `jina-core` is tested with the level of the PR

Comment (deepankarm) [reply tests/integration/jinad/Dockerfiles/Dockerfile:1]: [This](https://github.com/jina-ai/jina/blob/jinad-tests-branch/tests/integration/jinad/Dockerfiles/Dockerfile) pulls jinaai/jinad & just reinstalls jina from latest code, then restarts jinad

**Review by deepankarm — APPROVED** [review]

**Comment by deepankarm** [comment]
@hanxiao this needs your approval as code owner

**Review by hanxiao — APPROVED** [review]

**Pull request merged**
